Makefile

```diff
@@ -17,6 +20,14 @@ test_unit:
 test_component:
 	py.test $(PYTEST_OPTS) tests/component
 
+test_integration:
```

Comment:
I've tried to execute the tests on my local machine.
- i created fresh virtualenv
- installed requirements.txt
- installed requirements-dev.txt
- installed celery-dyrygent `python setup.py develop`

Then I did `make test_integration` and I get
```
E           celery.exceptions.ImproperlyConfigured: You need to install the redis library in order to use the Redis result store backend.
```

It would be good to add this requirement somewhere

Reply:
moved all to Makefile